jenkins/release-commit-lock/release-commit-lock.jenkinsfile

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+lib = library(identifier: 'jenkins@6.4.0', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+pipeline {
+    options {
+            timeout(time: 2, unit: 'HOURS')
+    }
+    agent {
+        docker {
+            label 'Jenkins-Agent-AL2023-X64-C54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v3'
+            registryUrl 'https://public.ecr.aws/'
+            alwaysPull true
+        }
+    }
+    parameters {
+        string(
+            name: 'RELEASE_VERSION',
+            description: 'The Release Version.',
+            trim: true
+        )
+        string(
+            name: 'OPENSEARCH_RELEASE_CANDIDATE',
```

Comment:
```suggestion
            name: 'OPENSEARCH_RELEASE_CANDIDATE_BUILD_NUMBER',
```
or 
```
name: 'OPENSEARCH_RC_BUILD_NUMBER',
```

Reply:
Are those not too long ?
`OPENSEARCH_RELEASE_CANDIDATE_BUILD_NUMBER` `OPENSEARCH_DASHBOARDS_RELEASE_CANDIDATE_BUILD_NUMBER`
The description already talks about build number

Reply:
Length I dont think it is an issue. It is just a var name here.